lightning/restore/restore.go

```diff
@@ -460,15 +460,21 @@ func (rc *RestoreController) listenCheckpointUpdates() {
 	rc.checkpointsWg.Done()
 }
 
-func (rc *RestoreController) runPeriodicActions(ctx context.Context, stop <-chan struct{}) {
-	switchModeTicker := time.NewTicker(rc.cfg.Cron.SwitchMode.Duration)
+func (rc *RestoreController) runPeriodicActions(ctx context.Context, stop <-chan struct{}, shouldSwitchMode bool) {
```

Comment:
i prefer removing the `shouldSwitchMode` parameter and check `rc.cfg.TikvImporter.Backend` inside this function